### ISSUE TITLE
Doesn't work with touchscreen

### ISSUE THREAD

**Issue by androw** [raphamorim/rio#225]
I'm using rio on openSUSE linux.

I'm unable to select text on the terminal by using the touchscreen.
This is working fine on alacritty.

This is a low priority issue I guess.

**Comment by raphamorim** [comment]
Oh yes, Rio doesn't have support for touch events but I would be super keen to add. I don't have any touchscreen device at the moment so will take a moment to look it.

If you want (or anyone) that would be keen to create a PR adding support, can follow up Alacritty touch strategy https://github.com/alacritty/alacritty/blob/34b5dbacd28cd1abaedf1d81cc0ebe57aa44a086/alacritty/src/input.rs#L801-L911

**Comment by androw** [comment]
I'll try to look into it

**Comment by raphamorim** [comment]
Also connected to the link I've sent https://github.com/alacritty/alacritty/blob/master/alacritty/src/event.rs#L1297

**Comment by raphamorim** [comment]
Closing it after #226 implementation
